src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1228,33 +1227,14 @@ export class ResultsViewPageStore {
         }
     }, []);
     
-    //user saved virtual studies
-    private readonly virtualStudies = remoteData(sessionServiceClient.getUserVirtualStudies(), []);
-    
-    private readonly virtualStudyIdToStudy = remoteData({
-        await: ()=>[this.virtualStudies],
-        invoke: async ()=>{
-            return _.keyBy(
-                this.virtualStudies.result.map(virtualStudy=>{
-                    let study = {
-                        allSampleCount:_.sumBy(virtualStudy.data.studies, study=>study.samples.length),
-                        studyId: virtualStudy.id,
-                        name: virtualStudy.data.name,
-                        description: virtualStudy.data.description,
-                        cancerTypeId: "My Virtual Studies"
-                    } as CancerStudy;
-                    return study;
-                }), x =>x.studyId);
-        }
-    },{});
-
     //this is only required to show study name and description on the results page
     readonly queriedStudies = remoteData({
```

Comment:
there's something weird about the name of this.  queriedStudies suggests relationship with the studies collection, but in fact is just returning contents of cohortIdsList, should there be anything inside of it.   Seems to the the name queriedStudies needs to be more descriptive.